### ISSUE TITLE
503 Service Unavailable errors from registry.npmjs.org

### ISSUE THREAD

**Issue by MoisesFlores22** [npm/registry-issue-archive#349]
When you do `yarn` or `npm install` it throws a Request failed \"503 Service Unavailable\""

**Comment by HamidAghdaee** [comment]
This was working for us for a few minutes. down again

**Comment by MoisesFlores22** [comment]
I think is kind of intermitent

**Comment by manatarms** [comment]
Any updates on this? We're also hitting the same issue.

**Comment by itschmp** [comment]
I have this same issue too. Anyone got some workaround solutions?

**Comment by jsonbecker** [comment]
We have been having trouble for two days now with intermittent CI failures from this.

**Comment by iam4x** [comment]
We're having same issues for two days now from our CI, but https://status.npmjs.org/ does not reflect the failures.

**Comment by drazisil** [comment]
This issue from a few days ago seems relevant, given the errors appear to be the same. https://github.com/npm/registry/issues/339

edit: Direct link to status: https://status.npmjs.org/incidents/qxj3kdzw4jrg

**Comment by exAspArk** [comment]
The [https://status.npmjs.org/](https://status.npmjs.org/) page was finally updated:

![image](https://user-images.githubusercontent.com/577441/41049887-ea22d46e-697f-11e8-8108-c3301036f6be.png)

**Comment by cweekly** [comment]
I originally posted https://github.com/npm/npm/issues/20904 and there's been no improvement...
at least the [status page](https://status.npmjs.org/) has been updated. 

Private registries FTW? Sigh.

**Comment by propattern** [comment]
Yes for a good part of the day this issue was affecting all our builds.
We also tried `yarn config set registry 'https://registry.npmjs.org'` and that didn't help.
Within the last hour, the issue seems to have been resolved.

**Comment by cweekly** [comment]
@propattern the problem is with the registry.npmjs.org service. 
The yarn registry is https://registry.yarnpkg.com/ and even tho it's a proxy for the canonical npmjs.org reg, AFAICT yarnpkg.com has acted as a reliable cache during this service disruption.  
FWIW I tried setting .npmrc with registry=https://registry.yarnpkg.com/ and (possibly coincidentally) haven't seen a 503 since. 

Appveyor has been really slow for us though -- likely bc people are freaking out and blindly hammering on CI builds.....

**Comment by drazisil** [comment]
@cweekly Per https://status.npmjs.org/incidents/qxj3kdzw4jrg this should be resolved. The majority of complaints I saw for this were against the Yarn registry, so I'm somewhat confused when you say they had no issues.

**Comment by cweekly** [comment]
@drazisil Interesting. FWIW, wasn't claiming yarn reg had no issues, just that I hadn't seen any. (shrug) Thanks and good luck and have a nice day. :)

**Comment by drazisil** [comment]
Sorry I misunderstood @cweekly . Same to you :)

**Comment by cweekly** [comment]
P.S. FTR I just saw this issue again, 23 min ago.
So the mitigation in https://status.npmjs.org/incidents/qxj3kdzw4jrg might not have done the trick...

**Comment by 0xadada** [comment]
working for me now 🎉

**Comment by dylanbutler** [comment]
This has been happening for us often over the past 2 days. Can we please look into this?
